integration/spark2/src/main/scala/org/apache/spark/sql/execution/strategy/CarbonLateDecodeStrategy.scala

```diff
@@ -479,6 +479,21 @@ private[sql] class CarbonLateDecodeStrategy extends SparkStrategy {
       case a: Attribute if isComplexAttribute(a) => a
     }.size == 0 )
 
+    // block filters for lucene with more than one text_match udf
+    // Todo: handle when lucene and normal query filter is supported
+    if (predicates.nonEmpty) {
+      if (predicates.seq.head.isInstanceOf[ScalaUDF]) {
```

Comment:
Logic seems wrong, supposed to be check over all predicates not just on head

Reply:
Now..filter on text_match and normal query is not supported(TEXT_MATCH('name:n') AND id=5). Hence for lucene ,filter should be explicitily lucene text_match.

Reply:
@ravipesala please review